Makefile

```diff
@@ -14,19 +15,21 @@ clean: ## remove build artifacts
 	rm -rf samples/css/
 	rm -rf samples/edx-bootstrap/
 
-samples: clean ## build the samples
+build: clean ## build the npm package
+
+build.samples: clean ## build the samples
 	mkdir samples/edx-bootstrap
 	cp -r sass/ samples/edx-bootstrap/sass
 	mkdir samples/css
 	mkdir samples/css/edx
 	mkdir samples/css/open-edx
 	./node_modules/node-sass/bin/node-sass samples/edx/sass --output samples/edx/css --include-path samples --include-path node_modules
 	./node_modules/node-sass/bin/node-sass samples/open-edx/sass --output samples/open-edx/css --include-path samples --include-path node_modules
-	./node_modules/.bin/opn samples/index.html
 
-build: clean ## build the npm package
+samples: clean build.samples ## build and show the samples
+	./node_modules/.bin/opn samples/index.html
 
-preview: samples ## build the preview site
+preview: build.samples ## build the preview site
```

Comment:
I wonder if this one should open the preview URL, to mirror what `samples` does.